JM/index.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+app.post('/addPerson', function(postmanRequest, postmanresponse){
+    console.log(postmanRequest.body)
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson', postmanRequest.body) 
+    .then(function(apiResopnse){
+        postmanresponse.json({"message": "User added"})
+    })
+})
+
+app.get('/getAllPeople', function(postmanRequest, postmanresponse){
+    const url = 'http://java-sample-api-2020.herokuapp.com/getAllPeople'
+    data = axios.get(url)
```

Comment:
I think that the data variable here would be equal to a promise. Id recommend looking at and understanding the [example get route](https://github.com/Codubee/Node-Js-API-Examples/blob/axios-examples/index.js)

Reply:
I believe that this is still incorrect Jaime. Lets setup some time so I can explain what is going on